### ISSUE TITLE
system requirements for training

### ISSUE THREAD

**Issue by ogulcankarayel5** [AlexeyAB/darknet#4566]
I have nvidia geforce 920m 2gb ,8gb ram and i3 5005U CPU. Can I train with 10.000 images

**Comment by AlexeyAB** [comment]
Yes, even with 10 000 000 images, but set `batch=64 subdivisions=64 random=0` in your cfg-file.

**Comment by oziblack05** [comment]
How long training will take ? For average

**Comment by AlexeyAB** [comment]
yolov3-tiny-prn.cfg for 10 000 iterations will take about 1 week on geforce 920m

**Comment by ogulcankarayel5** [comment]
Is there any way to reduce time
